src/pages/workspace/accounting/xero/export/XeroExportConfigurationPage.tsx

```diff
@@ -0,0 +1,111 @@
+import type {WithPolicyConnectionsProps} from '@pages/workspace/withPolicyConnections';
+import withPolicyConnections from '@pages/workspace/withPolicyConnections';
+import CONST from '@src/CONST';
+
+type MenuItem = MenuItemProps & {pendingAction?: OfflineWithFeedbackProps['pendingAction']};
+
+function XeroExportConfigurationPage({policy}: WithPolicyConnectionsProps) {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+    const policyID = policy?.id ?? '';
+    const policyOwner = policy?.owner ?? '';
+    const {export: exportConfiguration, errorFields, pendingFields} = policy?.connections?.xero?.config ?? {};
+    const menuItems: MenuItem[] = [
+        {
+            description: translate('workspace.xero.preferredExporter'),
+            onPress: () => {},
+            brickRoadIndicator: errorFields?.exporter ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined,
+            title: exportConfiguration?.exporter ?? policyOwner,
+            pendingAction: pendingFields?.export,
+            error: errorFields?.exporter ? translate('common.genericErrorMessage') : undefined,
+        },
+        {
+            description: translate('workspace.xero.exportExpenses'),
+            title: translate('workspace.xero.purchaseBill'),
+            interactive: false,
+            shouldShowRightIcon: false,
+            helperText: translate('workspace.xero.exportExpensesDescription'),
+        },
+        {
+            description: translate('workspace.xero.purchaseBillDate'),
+            onPress: () => {},
+            brickRoadIndicator: errorFields?.billDate ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined,
+            title: exportConfiguration?.billDate,
+            pendingAction: pendingFields?.export,
+            error: errorFields?.billDate ? translate('common.genericErrorMessage') : undefined,
+        },
+        {
+            description: translate('workspace.xero.exportInvoices'),
+            title: translate('workspace.xero.salesInvoice'),
+            interactive: false,
+            shouldShowRightIcon: false,
+            helperText: translate('workspace.xero.exportInvoicesDescription'),
+        },
+        {
+            description: translate('workspace.xero.exportCompanyCard'),
+            title: translate('workspace.xero.bankTransactions'),
+            shouldShowRightIcon: false,
+            interactive: false,
+            helperText: translate('workspace.xero.exportDeepDiveCompanyCard'),
+        },
+        {
+            description: translate('workspace.xero.xeroBankAccount'),
+            onPress: () => {},
+            brickRoadIndicator: errorFields?.nonReimbursableAccount ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined,
+            title: undefined,
+            pendingAction: pendingFields?.export,
+            error: undefined,
+        },
+    ];
+
+    return (
+        <AccessOrNotFoundWrapper
```

Comment:
Can we not use ConnectionLayout here ?

Reply:
ConnectionLayout adds extra padding to MenuItems, but not to the page description.

I tried fixing it, but it breaks other pages without MenuItem. 

I think that we can have another PR that fixes the component. And refactors  all pages to use it

Reply:
Sure. I have the same issue where I am using SelectionList, it has extra padding.  If we are planning to fix as a follow up, I can revert the change in two pages where I am having padding issues.

Reply:
@mananjadhav sounds good!